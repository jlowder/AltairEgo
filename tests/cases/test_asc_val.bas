10 A$ = "A"
20 PRINT "ASC of 'A':", ASC(A$)
30 B$ = "123"
40 PRINT "VAL of '123':", VAL(B$)
50 C$ = "45.67"
60 PRINT "VAL of '45.67':", VAL(C$)
70 END
