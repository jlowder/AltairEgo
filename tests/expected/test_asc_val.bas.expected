Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
ASC of 'A':    65
VAL of '123':  123
VAL of '45.67':              45.67
OK
